helm/trino-gateway/templates/deployment.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "trino-gateway.fullname" . }}
+  labels:
+    {{- include "trino-gateway.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "trino-gateway.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+          # Include the version of the configSecret as an input to the deployment checksum. This causes pods to restart on helm update
+          # if the secret is updated. Helm template must be run with the --dry-run=server option to prevent a nil pointer
+          checksum/config: {{ (lookup "v1" "Secret" .Release.Namespace "config-yaml").metadata.resourceVersion | sha256sum}}
+          # checksum/config:  (lookup "v1" "Secret" .Release.Namespace $.Values.configSecret).metadata.resourceVersion | sha256sum
+      {{- with .Values.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "trino-gateway.labels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "trino-gateway.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+              - "java"
+              - "--add-opens=java.base/java.lang=ALL-UNNAMED"
+              - "--add-opens=java.base/java.net=ALL-UNNAMED"
+              - "-jar"
+              - "/usr/lib/trino/gateway-ha-7-jar-with-dependencies.jar"
```

Comment:
this will be changed to `"/usr/lib/trino/gateway-ha-jar-with-dependencies.jar"` once an image is published for release 8

Reply:
can you change that now so the chart will work for 8 and onwards